src/core/client/stream/tabs/comments/views/reportComment/components/ReportCommentForm.tsx

```diff
@@ -0,0 +1,220 @@
+import { Localized } from "fluent-react/compat";
+import { get } from "lodash";
+import React, { StatelessComponent } from "react";
+import { Field, Form } from "react-final-form";
+
+import { OnSubmit } from "talk-framework/lib/form";
+import { validateMaxLength } from "talk-framework/lib/validation";
+import {
+  Button,
+  Flex,
+  HorizontalGutter,
+  RadioButton,
+  Typography,
+} from "talk-ui/components";
+
+import { PropTypesOf } from "talk-ui/types";
```

Comment:
Review import ordering, a bit off here and there.. Is there a tslint linting package we can use to help with this??

Reply:
Couldn't find one unfortunately :-(